src/xcode/ENA/ENA/Source/Scenes/TicketValidation/CertificateSelection/TicketValidationCertificateSelectionViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class TicketValidationCertificateSelectionViewController: DynamicTableViewController, DismissHandling {
```

Comment:
why is the protocol `dismissHandling` required?
- if so you should call `dismiss()` inside `wasAttemptedToBeDismissed` and use the `dismissHandlingCloseBarButton` for close button
- otherwise this can get removed?